sdk/program/src/message/mod.rs

```diff
@@ -0,0 +1,28 @@
+//! A library for generating a message from a sequence of instructions
+
+mod original;
+mod v0;
+mod versions;
+
+pub use original::Message;
+
+pub const MESSAGE_HEADER_LENGTH: usize = 3;
+
+#[frozen_abi(digest = "BVC5RhetsNpheGipt5rUrkR6RDDUHtD5sCLK1UjymL4S")]
```

Comment:
related to this (https://github.com/solana-labs/solana/pull/18725/files#r672288029), could you stop carrying over my bad code by removing this redundant `frozen_abi`?

Reply:
Removed!